Branch-SDK/src/io/branch/indexing/ContentDiscoverer.java

```diff
@@ -190,16 +198,11 @@ private void discoverFilteredViewContents(JSONArray viewIDArray, JSONArray conte
 
     private void discoverViewContents(ViewGroup viewGroup, JSONArray contentDataArray, JSONArray contentKeysArray, Resources res, boolean isClearText) {
         for (int i = 0; i < viewGroup.getChildCount(); i++) {
-            
             View childView = viewGroup.getChildAt(i);
-            if ((childView.getVisibility() == View.VISIBLE) && (childView instanceof ViewGroup)) {
```

Comment:
Why we need this change? This caused an error on content scraping and fixed in a previous PR